datasets/Binford/societies.csv

```diff
@@ -337,4 +337,4 @@ B215,xd995,Spring Valley Shoshoni,west2622,Spring Valley Shoshoni (B215),Spring
 B224,xd995,Antelope Valley Shoshoni,west2622,Antelope Valley Shoshoni (B224),"Spring Valley, Antelope Valley Shoshoni",1860,,,40.19,-114.24,40.19,-114.24,Original,
 B272,xd997,Chehalis,lowe1427,Chehalis (B272),,1880,,,46.87,-123.67,46.87,-123.67,Original,
 B293,xd998,Gitxsan,gitx1241,Gitksan (B293),Gitksan,1880,,,55.65,-128.06,55.65,-128.06,Original,
-B294,xd999,Koniag,koni1262,Konaig (B294),,1840,,,57.93,-153.16,57.93,-153.16,Original,
+B294,xd999,Konaig,koni1262,Konaig (B294),,1840,,,57.93,-153.16,57.93,-153.16,Original,"1Oct2018 - change to dialect and language mapping from beri1255 (no longer exists) to koni1262 (dialect of language paci1278) following update to Glottolog version 3.3.2. Note, this change may not be incorporated to Bouckaert/Atkinson global tree because of timing of correction."
```

Comment:
@kirbykat Is this a regression or do you think HH's suggestion of Konaig being a miss-typed Koniag is wrong?

Reply:
@xrotwang - I think this is a mistake on my end, thanks for flagging it. It should be Koniag.

Reply:
@xrotwang - by the way I just did a search and replace in my CSV file, and "Konaig" appears a couple of times in the comment field. Can a search and replace be quickly implemented across this entire file?

Reply:
Change in all 3 places:
```bash
$ grep "Konaig" datasets/*/societies.csv
datasets/Binford/societies.csv:B297,xd1027,Chugach,chug1254,Chugash (B297),"Chugachigmiut, Chugash",1930,South Alaskan Eskimo (NA10),http://ehrafworldcultures.yale.edu/collection?owc=NA10,61.21,-147.61,61.21,-147.61,Original,"3Oct2018 Assigned to dialect chug1254 (language does not change) to help differentiate this society from the closely related Konaig, also now mapped to language paci1278."
datasets/Binford/societies.csv:B294,xd999,Konaig,koni1262,Konaig (B294),,1840,,,57.93,-153.16,57.93,-153.16,Original,"1Oct2018 - change to dialect and language mapping from beri1255 (no longer exists) to koni1262 (dialect of language paci1278) following update to Glottolog version 3.3.2. Note, this change may not be incorporated to Bouckaert/Atkinson global tree because of timing of correction."
datasets/EA/societies.csv:Na10,xd1027,Chugach,chug1254,Chugach (Na10),"Chugachigmiut, Chugash",1930,South Alaskan Eskimo (NA10),http://ehrafworldcultures.yale.edu/collection?owc=NA10,60,-147,60.72,-146.49,Revised,"3Oct2018 Assigned to dialect chug1254 (language does not change) to help differentiate this society from the closely related Konaig, also now mapped to language paci1278."
```